airflow/www/static/js/dag/details/index.tsx

```diff
@@ -336,6 +362,17 @@ const Details = ({
               />
             </TabPanel>
           )}
+          {showXcom && run && (
```

Comment:
If we make the changes to our boolean vars, we don't need to check for `run` here